app/tests/unit/date-input.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { shallowMount, Wrapper } from '@vue/test-utils'
+import DateInput from '@/components/common/DateInput.vue'
+
+describe('DateInput.vue', () => {
+  let wrapper: Wrapper<Vue>
+
+  beforeEach(() => {
+    const label = 'Date Label'
+    const id = 'dateInputId'
+    const required = true
+    const value = '2022-12-30'
+    const maxDate = '2022-12-31'
+    const minDate = '2021-01-01'
+
+    wrapper = shallowMount(DateInput, {
+      propsData: { label, value, id, required, maxDate, minDate }
+    })
+  })
+
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders label and input with correct attributes', async () => {
+    await wrapper.vm.$nextTick()
```

Comment:
What is the purpose of `vm.$nextTick()` here?

Reply:
Removed it in the recent commit.